src/main/java/org/skypro/exam/controller/JavaQuestionController.java

```diff
@@ -0,0 +1,38 @@
+package org.skypro.exam.controller;
+
+import org.skypro.exam.model.Question;
+import org.skypro.exam.service.JavaQuestionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping
+public class JavaQuestionController {
+    private final JavaQuestionService javaQuestionService;
+
+    @Autowired
+    public JavaQuestionController(JavaQuestionService javaQuestionService) {
```

Comment:
используем плюсы полиморфизма
```suggestion
    public JavaQuestionController(QuestionService javaQuestionService) {
```